src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingWeightDisplay.kt

```diff
@@ -150,10 +150,6 @@ class FarmingWeightDisplay {
             if (weight == -1.0) {
                 if (!isLoadingWeight) {
                     val localProfile = HypixelData.profileName
-                    if (localProfile == "") {
```

Comment:
Why removing this? I have added this since some users were having crashes without.

Reply:
localProfile is blank when first logging into hypixel
and if the localProfile is blank, it will show the weight for the selected profile according to the elite api